modules/ROOT/pages/dataweave-cookbook-defaults.adoc

```diff
@@ -15,48 +17,50 @@ include::partial$dataweave1-links.adoc[tag=dataweave1Examples]
 
 == Using the Key Word default
 
-This example uses `default` to set a default value for a payload in case
-the payload value is `null`.
+Consider a Mule application that expects a JSON input with fields `id` and `name` to do a transformation. +
+You can configure default values for these fields in case the payload value is `null`. For example:
 
-.DataWeave Script:
 [source,DataWeave, linenums]
 ----
 %dw 2.0
 output application/json
 ---
-
-(payload.someField default "my default value")
+{
+  "userId": payload.id default "0000",
+  "userName": payload.name default "Undefined"
+}
 ----
 
-.Output
-[source,JSON,linenums]
+* If the Mule application receives the following JSON input:
++
+[source,json, linenums]
 ----
-"my default value"
+{
+  "id": "123",
+  "name": "Max the Mule"
+}
 ----
-
-You can copy and paste the following example into a Transform Message component.
-It uses the variable `myNullExample` to provide sample input.
-
-.DataWeave Script:
-[source,DataWeave, linenums]
++
+The DataWeave transformation output is:
```

Comment:
See suggestions above.

Also, GLOBALLY do not break an implied "if...then" sentence around an example (or break ANY sentence around ANY kind of graphic display. In this case, "*If* the Mule application receives the following JSON input:" <example> "[implied *then*]The DataWeave transformation output is:" <example>